Packs/CommonScripts/Scripts/IsIPPrivate/IsIPPrivate.py

```diff
@@ -0,0 +1,59 @@
+from ipaddress import IPv4Address, IPv4Network
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def check_ip_internal(ip, ranges):
```

Comment:
type hints make maintenance easier, consider adding them (here, and throughout the file)

Reply:
I pushed the changes, maybe you're looking at an old commit? This is the commit where I fixed it to be one line: https://github.com/demisto/content/pull/24077/commits/2a787c8cf3c8866d47c780bee3f3c3237970ac9d

Regarding the type hints if it's OK I would prefer not to add them at this point